### DISCUSSION TITLE
adjust homing speed

### DISCUSSION THREAD

**Discussion by mimes123** [mriscoc/Ender3V2S1#808] [Q&A]
Hi I often see that the z-offset should be changed after homing. 
everything is tight and no lose screws.

I have a theory that says me it is because it is homing too fast down on the end switch.

is there a way how I can control the speed so it moves slower before it hits the end switch ?

**Comment by mimes123** [comment]
i am still running the 2022 release by the way

**Comment by mriscoc** [comment]
You need to compile your own custom version to be able to change the home axes speed.
